tools/StaticAnalysis/Exceptions/Az.Compute/BreakingChangeIssues.csv

```diff
@@ -0,0 +1,6 @@
+AssemblyFileName,ClassName,Target,Severity,ProblemId,Description,Remediation
+Az.Compute,Microsoft.Azure.Commands.Compute.NewAzureAdditionalUnattendContentCommand,Add-AzVMAdditionalUnattendContent,0,3010,The property 'PublicIPAddressSkuName' of type 'Microsoft.Azure.Management.Compute.Models.PublicIPAddressSku' has been removed.,Add the property 'PublicIPAddressSkuName' back to type 'Microsoft.Azure.Management.Compute.Models.PublicIPAddressSku'.
+Az.Compute,Microsoft.Azure.Commands.Compute.NewAzureAdditionalUnattendContentCommand,Add-AzVMAdditionalUnattendContent,0,3010,The property 'PublicIPAddressSkuTier' of type 'Microsoft.Azure.Management.Compute.Models.PublicIPAddressSku' has been removed.,Add the property 'PublicIPAddressSkuTier' back to type 'Microsoft.Azure.Management.Compute.Models.PublicIPAddressSku'.
+Az.Compute,Microsoft.Azure.Commands.Compute.NewAzureAdditionalUnattendContentCommand,Add-AzVMAdditionalUnattendContent,0,3000,The type of property 'SecurityType' of type 'Microsoft.Azure.Management.Compute.Models.SecurityProfile' has changed from 'System.Nullable`1[Microsoft.Azure.Management.Compute.Models.SecurityTypes]' to 'System.String'.,Change the type of property 'SecurityType' back to 'System.Nullable`1[Microsoft.Azure.Management.Compute.Models.SecurityTypes]'.
+Az.Compute,Microsoft.Azure.Commands.Compute.SetAzureVMSecurityProfile,Set-AzVMSecurityProfile,0,2020,The cmdlet 'Set-AzVMSecurityProfile' no longer supports the type 'Microsoft.Azure.Management.Compute.Models.SecurityTypes' for parameter 'SecurityType'.,Change the type for parameter 'SecurityType' back to 'Microsoft.Azure.Management.Compute.Models.SecurityTypes'.
+Az.Compute,Microsoft.Azure.Commands.Compute.Automation.SetAzureRmVmssSecurityProfileCommand,Set-AzVmssSecurityProfile,0,2020,The cmdlet 'Set-AzVmssSecurityProfile' no longer supports the type 'Microsoft.Azure.Management.Compute.Models.SecurityTypes' for parameter 'SecurityType'.,Change the type for parameter 'SecurityType' back to 'Microsoft.Azure.Management.Compute.Models.SecurityTypes'.
```

Comment:
Changing enum to string should not be a breaking change, but removal of properties may cause user's script to break, so they should not be suppressed.

Reply:
the first two ones are not exactly removed, but 'PublicIpAddressesSkuName' and 'PublicIPAddressSkuTier' are changed to 'name' and 'tier'.    But there are no cmdlets right now that use those models.

Reply:
Okay in that case they don't count as breaking change